core/sql/sqlcomp/CmpDescribe.cpp

```diff
@@ -3059,6 +3061,36 @@ short CmpDescribeSeabaseTable (
 
       outputLongLine(*space, viewtext, 0);
 
+      //display comment for VIEW
+      if (objectUID > 0/* && COMMENT_CQD*/)
+        {
+          if (cmpSBD.switchCompiler())
+            {
+              *CmpCommon::diags() << DgSqlCode(-CAT_UNABLE_TO_RETRIEVE_PRIVS);
+              return -1;
```

Comment:
Is this the correct SQL code?  Is COMMENT_CQD valid?

Reply:
I add a new error code CAT_UNABLE_TO_RETRIEVE_COMMENTS=1033, is this a good way?